### PR TITLE
Edit documentation

### PR DESCRIPTION
**Pull request by minalike**
This PR is a proofreading and light editing of the documentation for ClassyConfig.

Pages reviewed:

- [x] Getting Started
- [x] Load Your Config
- [x] Using Multiple Configs
- [ ] Conflicting Configs
- [ ] Custom Loaders
- [ ] Reference

Notes:
- Double quotes were more prevalent in the examples and so single quotes were converted to match
- Some examples are documented on the repo as well and should also be updated once satisfied with the changes (e.g.: [using multiple configs](https://github.com/GDWR/classy-config/tree/main/examples/multiple_configs))

Suggestions:
- It would be nice to capitalize the word "documentation" in the page title and banner
- Perhaps a link to the source code repository in the footer

### CODE REVIEW

**Comment by GDWR** [comment]
@minalike Is it okay for me to review your current changes and look for them to be merged into `main`? The additional pages you have listed you would review can be done in another PR.

**Comment by minalike** [comment]
Yes please go ahead.

File: docs/source/guides/custom-loaders.rst [hunk 1]
```diff
@@ -29,5 +29,5 @@ Here is an example of creating a simple config loader for ``.txt`` files.
     register_loader(".txt", txt_loader)
     register_config("config.txt")
 
-    print(f"data: {ConfigValue('data', int)}")
-    print(f"name: {ConfigValue('name', str)}")
+    print(f"data: {ConfigValue("data", int)}")
+    print(f"name: {ConfigValue("name", str)}")
```
Comment (GDWR) [inline docs/source/guides/custom-loaders.rst:33]: The internal single quote was used on purpose, it is required to use the alternative quote for strings inside f-strings.

**Review by GDWR — CHANGES_REQUESTED** [review]
One change, after that I will merge into main. Cheers.

**Pull request merged**